Add Iris Navarrete advice file

IrisN94 opened the pull request.

twosavoie commented:
Hey @IrisN94,

Great job adding your advice file to the advice folder!

The next step would be to add a link in the index.html file to your advice file. Be sure the spelling of the file name is exactly the same as the file's name. (Changing the file name to lowercase would be a good idea. 😄)

Once you fix that up, you can add, commit, and push and your changes will join your pull request here.

Let us know if you need any help! You've got this!

Best,
Lisa

IrisN94 commented:
Hello Lisa, thank you for the direction. But i went back and looked at the link I added on the index.html page and the link I added looks correct and I pushed it. I am not sure what else I am supposed to do, sorry I am a bit lost.

twosavoie commented:
Hi @IrisN94,

Looking at your advice branch's index.html file (https://github.com/IrisN94/git-practice/blob/advice/index.html), I don't see a link to your advice file (advice/Iris-Navarrete-advice.html). Do you see it on the local version of your code? When you see it, can you check to be sure you're on the advice branch? And can you run **git status** to be sure that all changes have been added, committed, and pushed to GitHub? 

Let us know and we'll go from there. 🙂 

Best,
Lisa

twosavoie commented:
Hey @IrisN94,

We haven't heard from you in a while, so we're going to go ahead and close this pull request. Feel free to submit a new one when you're ready!

Best,
Lisa
